corehq/apps/reports/tests/test_generic.py

```diff
@@ -74,3 +80,95 @@ def test_handles_rows(self):
         self.assertEqual(result[0], ['One', 'Two'])
         self.assertEqual(result[1], ['Three', 'Four'])
         self.assertEqual(result[2], ['Five', 'Six'])
+
+
+class GetSerializableReportParameters(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = WebUser(domain='test-domain', username='test-user')
+        cls.user.save()
+        cls.addClassCleanup(cls.user.delete, deleted_by_domain='test-domain', deleted_by=None)
+        cls.report = object.__new__(get_report_class('daily_form_stats'))
```

Comment:
I think we can avoid the indirection here. We know what class `daily_form_stats` is associated with, right?